source/scss/components/_design.scss

```diff
@@ -1,39 +1,24 @@
 .designers__section {
   display: flex;
   justify-content: space-between;
-  font-family: Poppins;
   margin-bottom: 84px;
 }
 
 .designers__section-description {
   max-width: 486px;
-  font-family: Poppins;
-  font-style: normal;
-  font-weight: bold;
-  font-size: 16px;
-  line-height: 26px;
-  color: #2f281e;
+  @include text-formatter;
```

Comment:
with @include you still add the code to all places mixin is used
all font properties in CSS are inherited so that's why base font-family and font-size are usually set on body selector
then font properties are specified for other element if they have different styling
no need to add the same styles everywhere